ReactWindows/ReactNative.Shared/Bridge/ReflectionReactDelegateFactory.cs

```diff
@@ -163,20 +170,34 @@ private Extractor CreateExtractor(Type type, string moduleName, string methodNam
             var args = new object[extractors.Count];
             for (var j = 0; j < c; ++j)
             {
-                var result = extractors[j](reactInstance, jsArguments, idx);
-                args[j] = result.Value;
-                idx = result.NextIndex;
+                var extractorResult = extractors[j](reactInstance, jsArguments, idx);
```

Comment:
>extractorResult [](start = 20, length = 15)

Side note - we need to do the same trick for method extractors that we did for ViewManager props, namely instantiate default values for structs.